packages/paste-website/src/pages/components/tooltip/index.mdx

```diff
@@ -155,19 +161,24 @@ A tooltip **must** be placed on a **focusable** element, such as an anchor or bu
 </Box>`}
 </LivePreview>
 
-<Callout>
-  <CalloutTitle>Icons and tooltips</CalloutTitle>
-  <CalloutText>
-    If your tooltip wraps an element that includes only an icon, make sure you set the `decorative` prop to false, and
-    give the icon a title. The title of the icon should describe what the icon is representing, and the tooltip
-    describes what that means. This ensures the icon and tooltip ara accessible to screen readers.
-  </CalloutText>
-</Callout>
+### Adding a Tooltip to non-Paste React components
+
+If you want to add a Tooltip around a React component that isn't in the Paste library, you must make sure that component [forwards refs](https://reactjs.org/docs/forwarding-refs.html).
```

Comment:
Not sure we need to mention non-paste components here. This is equally relevant if you wrap a paste component, but don't forward refs to it.

Maybe it's your teams "Buy" button that is used everywhere, and you create a BuyButton component that composes our Button into primary and full width state. That would need a ref forwarded to it

Reply:
I had a hard time framing this in any other way that would be clear. Paste components `forwardRef` by default, so I wanted to describe what to look out for without having to explain/overwhelm by mentioning `forwardRef` in the heading.

Reply:
Yeah, it's not easy. I just don't feel like we should be documenting non-paste components outside of primitive pages.

Reply:
Improved. Lmk!